### PR TITLE
alert component added

### PR DESCRIPTION
**Pull request by Sharmaryan**
Screenshot (11)
Screenshot (12)

### CODE REVIEW

File: styles/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  color: rgb(138, 230, 126);
+  background-color: rgb(63, 146, 63);
+}
+.alert-danger {
+  color: rgb(243, 212, 212);
+  background-color: rgb(245, 115, 115);
+}
+.alert-primary-outline{
+    outline: 3px solid rgb(11, 230, 230);
+    color: rgb(11, 230, 230);
```
Comment (zabihshaik) [inline styles/alert/alert.css]: Better to use variables for color, if it is used repeatedly.

File: styles/alert/alert.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/style.css" />
+    <title>Document</title>
```
Comment (zabihshaik) [inline styles/alert/alert.html]: Add a title, for eg. Alert | your CL Name

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. Just a couple of comments. Plz check.

File: styles/alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  font-weight: bold;
+}
+
+.alert-primary {
+  color: rgb(11, 230, 230);
+  background-color: var(--default);
+}
+.alert-secondary {
+  color: rgb(165, 103, 11);
+  background-color: rgb(245, 245, 185);
```
Comment (Bhakti199) [inline styles/alert/alert.css]: define these colors in root first and then use it

**Review by Bhakti199 — APPROVED** [review]
looks cool, good to merge

File: styles/alert/alert.html (hunk 2 above)
Comment (Sharmaryan) [reply styles/alert/alert.html]: done

File: styles/alert/alert.css (hunk 3 above)
Comment (Sharmaryan) [reply styles/alert/alert.css]: done

File: styles/alert/alert.css (hunk 1 above)
Comment (Sharmaryan) [reply styles/alert/alert.css]: done

File: styles/alert/alert.css [hunk 4]
```diff
@@ -17,35 +23,34 @@
...
+  outline: 3px solid var(--dark-yellow);
+  color: var(--dark-yellow);
 }
-.alert-success-outline{
-    outline: 3px solid rgb(63, 146, 63);
-    color: rgb(63, 146, 63);
+.alert-success-outline {
+  outline: 3px solid var(--dark-green);
+  color: var(--dark-green);
 }
```
Comment (saadahmadhere) [inline styles/alert/alert.css:52]: space after each {} would make the code cleaner.

File: styles/alert/alert.html [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+          <p>A simple Secondary Alert</p>
+        </div>
+        <div class="alert alert-success">
+          <p>A simple Success Alert</p>
+        </div>
+        <div class="alert alert-danger">
+          <p>A simple Danger Alert</p>
+        </div>
+        <div class="inner-container-code">
+
```
Comment (saadahmadhere) [inline styles/alert/alert.html:57]: unwanted line space here

File: styles/alert/alert.html [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+          <p>A simple Secondary Alert</p>
+        </div>
+        <div class="alert alert-success-outline">
+          <p>A simple Success Alert</p>
+        </div>
+        <div class="alert alert-danger-outline">
+          <p>A simple Danger Alert</p>
+        </div>
+<div class="inner-container-code">
+
```
Comment (saadahmadhere) [inline styles/alert/alert.html:75]: here too

**Review by saadahmadhere — APPROVED** [review]
Code looks neat. Good to see the use of variables. Just have a look at the line spaces.

File: styles/alert/alert.css (hunk 4 above)
Comment (Sharmaryan) [reply styles/alert/alert.css:52]: thanks , I will do that

File: styles/alert/alert.html (hunk 5 above)
Comment (Sharmaryan) [reply styles/alert/alert.html:57]: will remove them

**Comment by preethasri** [comment]
Hey aryan the UI looks good and neat . Just make it responsive for mobile also  that would be better

**Pull request merged**